convert2rhel/pkghandler.py

```diff
@@ -594,13 +605,13 @@ def remove_pkgs_unless_from_redhat(pkgs_to_remove, backup=True):
     """
     if not pkgs_to_remove:
         loggerinst.info("\nNothing to do.")
-        return
+        return pkgs_to_remove
```

Comment:
I don't think I understood the reasoning behind this change. Why should we return the `pkgs_to_remove`? 

Instead, can't we return an empty dict, meaning that there was no package removed? The idea of this function is to return the packages that were removed, not the ones we asked for removal.

Reply:
This will return an empty list, so later a sort isn't done on a NoneType object

Reply:
Even though they're equivalent, I think @r0x0d 's question about why we're returning `pkgs_to_remove` here points out a readability problem.  Returning a new empty list won't be ay different in practice but it will be easier to understand that the empty list is different than the input.

Additionally, since pkgs_to_remove is a list, it is a mutable value.  Returning the input list as the output runs the risk that the caller will mutate one or the other after this function runs, not knowing that they're really the same list.  That could have unexpected consequences.  So it is better to return a new empty list for robustness as well.

Reply:
```suggestion
        return []
```

Reply:
Ok I will make this change